typescript/ws-protocol-examples/src/examples/service-server.ts

```diff
@@ -25,13 +29,56 @@ async function main(): Promise<void> {
   });
   setupSigintHandler(log, ws);
 
-  const service = {
-    name: "/set_bool",
-    requestSchema: "bool data",
-    responseSchema: "bool success\nstring message",
+  const serviceDefRos: Omit<Service, "id"> = {
+    name: "/set_bool_ros",
     type: "std_srvs/SetBool",
+    request: {
+      encoding: "ros1",
+      schemaName: "std_srvs/SetBool_Request",
+      schemaEncoding: "ros1msg",
+      schema: "bool data",
+    },
+    response: {
+      encoding: "ros1",
+      schemaName: "std_srvs/SetBool_Response",
+      schemaEncoding: "ros1msg",
+      schema: "bool success\nstring message",
+    },
   };
-  const serviceId = server.addService(service);
+  const serviceDefJson: Omit<Service, "id"> = {
+    name: "/set_bool_json",
+    type: "std_srvs/SetBool",
+    request: {
+      encoding: "json",
+      schemaName: "SetBoolJsonRequest",
+      schemaEncoding: "jsonschema",
+      schema: `{
```

Comment:
nit: you could do `schema: JSON.stringify({ ... })` and get some help from Prettier